index.js

```diff
@@ -26,7 +26,15 @@ app.get("/", (req,res)=>{ //lo que recibe/envia cuando se accede al nodo raíz:
 
 app.get(BASE_API_PATH +"/contacts", (req,res) =>{
     console.log(Date() + " - GET /contacts");
-    res.send([]);
+    db.find({}, (err, contats) =>{
+        if(err){
+            console.log(Date() + " - "+ err);
+            res.sendStatus(500);
+        }else{
+            res.send(contats);
```

Comment:
Please, change this code to remove the_ide from the information thats is sent to the user

Reply:
Changes done